tests/Fixtures/TestBundle/DataProvider/ContainNonResourceItemDataProvider.php

```diff
@@ -35,10 +35,10 @@ public function getItem(string $resourceClass, $id, string $operationName = null
     {
         // Retrieve the blog post item from somewhere
         $cnr = new ContainNonResource();
-        $cnr->id = $id['id'];
+        $cnr->id = $id;
```

Comment:
I reverted Antoine's changes in this custom dataprovider so it proves it will continue to work in behat tests "the old way"